preprocessing.py

```diff
@@ -29,4 +29,45 @@ def prepare_data(df):
     encoded_cat_features = transform.get_feature_names(cat_cols)
     df_enc = pd.DataFrame.sparse.from_spmatrix(encoded_cats, columns = encoded_cat_features)
 
-    return X.join(df_enc), y
\ No newline at end of file
+    return X.join(df_enc), y
+
+
+def prepare_data_no_standardizing(df):
```

Comment:
Want to just use the my preprocessing function instead? It also selects the features we are going to need.